src/components/Calendar/stylesheet.scss

```diff
@@ -88,15 +89,76 @@
         top: $day-height;
         bottom: 0;
         width: 1px;
-        background-color: $color-light;
+        transition: 1s ease;
       }
     }
   }
 
-  .meetings {
+  .meeting {
+    transition: 1s ease;
   }
 
   &.mobile {
     margin-right: 0;
   }
 }
+
+.App.light .Calendar {
+  border-bottom: 1px solid $color-light;
+  border-right: 1px solid $color-light;
+
+  &.preview {
+    box-shadow: inset -6px 0 5px -1px #aaaaaa;
+  }
+
+  .times .time {
+    &:after {
+      background-color: $color-light;
+    }
+  }
+
+  .days .day {
+    &:after {
+      background-color: $color-light;
+    }
+  }
+
+  .meeting {
+    box-shadow: 0px 10px 10px 0px #707070;
+  }
+}
+
+.App.dark .Calendar {
+  background-color: #303030;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.2);
```

Comment:
Avoid using hard-coded color values by creating a variable in `../../variables.scss`. Duplicates are never good.

Same goes to any other stylesheets.